### PR TITLE
Updated Debian jenkins config file for new jenkins version & fixed config for jenkins.enable_access_log == 'yes'

### PR DESCRIPTION
**Pull request by dosercz**
Fixed merge problem with empty else branch for setting ```JENKINS_ARGS```.
Updated Debian config file to latest version pro jenkins package (all formula features should be retained there).

### CODE REVIEW

File: jenkins/files/Debian/jenkins.conf [hunk 1]
```diff
@@ -1,38 +1,40 @@
...
-PIDFILE=/var/run/${NAME}/${NAME}.pid
+PIDFILE=/var/run/$NAME/$NAME.pid
 
 # user and group to be invoked as (default to jenkins)
 JENKINS_USER="{{ jenkins.user }}"
 JENKINS_GROUP="{{ jenkins.group }}"
 
 # location of the jenkins war file
-JENKINS_WAR=/usr/share/${NAME}/${NAME}.war
+JENKINS_WAR=/usr/share/$NAME/$NAME.war
```
Comment (myoung34) [inline jenkins/files/Debian/jenkins.conf]: why the change here? 
It's a common practice to put variables in curly braches for shell expansion as well as surround with quotes (missing here) to prevent issues with spaces

**Comment by dosercz** [comment]
My idea is to minimize diff with original conf file from package. That's the reason, why e.g. curly braces are not there - they are not in original conf file.
